### ISSUE TITLE
error in _wrapper_func document code

### ISSUE THREAD

**Issue by shengjh** [zilliztech/arctern#622]
**Branch**
master
https://github.com/zilliztech/arctern/blob/c713d9c555d466097b1ad57c5b54054bfbb2475a/python/arctern/_wrapper_func.py#L820-L834
**What's the problem**
I have run this document code , and i get different result from the document string.
The result should be
```
>>> rst
0         0.0
1    111226.3
dtype: float64
```

**Comment by shengjh** [comment]
We can use doctest  to avoid this problem. I will open a new issue to track doctest discussion.
pthon.org says: 
```
The doctest module searches for pieces of text that look like interactive Python sessions,
and then executes those sessions to verify that they work exactly as shown. …
```
more info: https://docs.python.org/3/library/doctest.html

**Comment by shengjh** [comment]
solved in #662
